rosdep/python.yaml

```diff
@@ -8768,6 +8768,17 @@ python3-pytest-mock:
       packages: [pytest-mock]
   rhel: ['python%{python3_pkgversion}-pytest-mock']
   ubuntu: [python3-pytest-mock]
+python3-pytest-order:
+  arch: [python-pytest-order]
+  debian: [python3-pytest-order]
+  gentoo: [dev-python/pytest-order]
+  nixos: [python3Packages.pytest-order]
+  opensuse: [python3-pytest-order]
```

Comment:
Note sure why the linter is failing for opensuse, as the URL lists support for `SUSE SLE-15-SP2`.

Reply:
I'd just remove it in this case.

Reply:
Removed.